wao/strategy_controller.py

```diff
@@ -0,0 +1,34 @@
+from wao.util import _perform_execute, _perform_back_test
+import threading
+from wao.config import Config
+import time
+
+
+class StrategyController:
```

Comment:
this shoudn't be visible, please merge develop to your branch

Reply:
I did merge it but still the same

Reply:
doesn't make sense, something is wrong in the conflict resolution - create a pr with only your changes

Reply:
I went to the repo in github.com and copied and pasted it exactly the way it is but I still got the same thing

Reply:
doesn't make sense, something is wrong in the conflict resolution - **create a new pr with only your changes**

an already existing file can not be an added change in git